encrypted-media-respec.html

```diff
@@ -456,16 +483,31 @@
               <li>
                 <p>A value that is fully cleared/clearable along with browsing data, such as cookies, after which it will be replaced by a value that is <a def-id="non-associable"></a> (not just <a def-id="non-associable-by-application"></a>), even by a central server such as an <a href="#privacy-individualization">individualization</a> server, AND one or more of the following:</p>
                 <ul>
-                  <li><p>No indelible value or distinctive identifier was involved in the generation of the value.</p></li>
+                  <li><p>No <a def-id="distinctive-permanent-identifier"></a> or distinctive identifier was involved in the generation of the value.</p></li>
                   <li><p>It is a random value generated <em>without</em> inputs from the system.</p></li>
                   <li><p>It is a value provided by a server without the use of or knowledge of another distinctive identifier.</p></li>
                 </ul>
               </li>
             </ul>
           </div>
-          <p class="note">The source of the identifier does not affect whether it is distinctive.
-            For example, an identifier that is permanently part of the client device, contained in the CDM, generated on the client, or generated as part of some individualization or other provisioning process is considered distinctive if it meets the criteria above.
+        </dd>
+
+        <dt>Use of Distinctive Identifiers and Distinctive Permanent Identifiers</dt>
+        <dd>
+          <p>
+            An implementation, configuration, instance, or object <dfn id="uses-distinctive-identifiers">uses Distinctive Identifier(s)</dfn> if, at any time during its lifetime or the lifetime of related such entities,
+            it exposes, even in encrypted form, on or more <a def-id="distinctive-identifier">Distinctive Identifier(s)</a>, information about them, or values derived from or otherwise related to them outside the client.
+            This includes but is not limited to providing such a value to the application and/or license, <a href="#privacy-individualization">individualization</a>, or other server.
+          </p>
+          <p>
+            An implementation, configuration, instance, or object <dfn id="uses-distinctive-permanent-identifiers">uses Permanent Distinctive Identifier(s)</dfn> if, at any time during its lifetime or the lifetime of related such entities,
+            it exposes, even in encrypted form, on or more <a def-id="distinctive-permanent-identifier-maybe-plural"></a>, information about them, or values derived from or otherwise related to them outside the client.
```

Comment:
From above:
>"information about them" is very broad: a boolean indicating whether one was present or not would qualify.

This was adapted from line 416 ("It, information about it, or values derived from or otherwise related to it are exposed, even in encrypted form, outside the client."), which has been been in the Distinctive Identifier definition for a long time. I debated all the parts, but being consistent seemed appropriate.

As for the Boolean, I think it could apply. I don't think we would want an application to be able to ask "Do you have a permanent Foo Key System key on your device?" without triggering some of the behaviors associated with these definitions.